examples/usecases/incremental-training-with-layer-freezing.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>Should we rename the dataset to day_1, day_2 and day_3 (and add that this is hypothetical)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>